src/components/transactions/ExecuteTxButton/index.tsx

```diff
@@ -21,7 +21,7 @@ const ExecuteTxButton = ({
   const txNonce = isMultisigExecutionInfo(txSummary.executionInfo) ? txSummary.executionInfo.nonce : undefined
   const isPending = useIsPending(txSummary.id)
 
-  const isNext = !!txNonce && !!safe.nonce && txNonce === safe.nonce
+  const isNext = typeof txNonce !== undefined && typeof safe.nonce !== undefined && txNonce === safe.nonce
```

Comment:
Good catch!
`typeof` always returns a string tho. You want a direct comparison like `txNonce !== undefined` instead.

Reply:
Also noticed that `safe.nonce` can't be undefined anymore since we added an initial value of -1 a while back so I removed that check